### ISSUE TITLE
Link to Reversals?

### ISSUE THREAD

**Issue by jSylvestre** [ucdavis/Sloth#369]
Should reversals and transactions that have reversals have a specific link/info in the details section of the details page?
(The related Transactions should show this, but that can have other related txns)

**Comment by srkirkland** [comment]
Yes, use `ReversalTransactionId` and `ReversalOfTransactionId`.  If either is not null, show link in the transaction details section above `RELATED TRANSACTIONS`.  

Something like "REVERSAL OF: <>Click here to view reversed transaction"
